### PR TITLE
put link to speciffic android plugin docs and removed unnecessary info

### PR DESCRIPTION
**Pull request by Plamen5kov**

### CODE REVIEW

**Comment by Plamen5kov** [comment]
related: https://github.com/NativeScript/android-runtime-docs/pull/14

File: plugins/plugins.md [hunk 1]
```diff
@@ -116,12 +105,13 @@ my-plugin/
...
 * `platforms\ios\build.xcconfig`: This file modifies the native iOS configuration of your NativeScript project such as native dependencies and configurations. For more information about the format of `build.xcconfig`, see [`build.xcconfig` file](#buildxcconfig-specification).
 * `platforms\ios\Podfile`: This file describes the dependency to the library that you want to use. For more information, see [the CocoaPods article](CocoaPods.md).
 
+> **IMPORTANT:** For more detailed information on the android plugin structure, [see here]({% slug android-plugins-infrastructure %}).
```
Comment (ikoevska) [inline plugins/plugins.md]: Android :D

**Comment by ikoevska** [comment]
:+1:

**Comment by ErjanGavalji** [comment]
:+1: after a rebase onto master

**Pull request merged**